src/frontends/tensorflow/src/CMakeLists.txt

```diff
@@ -7,20 +7,8 @@ ov_add_frontend(NAME tensorflow
                 FILEDESCRIPTION "FrontEnd to load and convert TensorFlow file format"
                 LINK_LIBRARIES openvino::util openvino::runtime::dev)
 
-# add object library used in tests for private transformations
-
-add_library(openvino_tensorflow_frontend_static_tests STATIC EXCLUDE_FROM_ALL
-    helper_transforms/block_lstm_replacer.cpp
-    helper_transforms/embedding_segments_feature_fusing.cpp
-    helper_transforms/gru_block_cell_replacer.cpp
-    pass/transpose_sinking.cpp
-    exception.cpp
-    openvino_conversions.cpp
-    utils.cpp)
-
-target_include_directories(openvino_tensorflow_frontend_static_tests
-    PUBLIC
-        ${CMAKE_CURRENT_SOURCE_DIR}
-    PRIVATE
-        $<TARGET_PROPERTY:openvino::runtime::dev,INTERFACE_INCLUDE_DIRECTORIES>
-        $<TARGET_PROPERTY:openvino::frontend::tensorflow,INTERFACE_INCLUDE_DIRECTORIES>)
+set(TARGET_NAME "${FRONTEND_NAME_PREFIX}tensorflow${FRONTEND_NAME_SUFFIX}")
+target_link_libraries(${TARGET_NAME} PRIVATE openvino::frontend::tensorflow_common::dev)
+target_include_directories(${TARGET_NAME}
+        PUBLIC
```

Comment:
`PRIVATE`? 
In this case we don't need `BUILD_INTERFACE` generator expression below.

Reply:
We would like to let those linking with tensorflow_frontend still be able to use all the public API that was available previously.

Reply:
Error because of this https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fbuild-windows-vs2019/detail/build-windows-vs2019/209880/pipeline

Reply:
We need to have a public dependency because it was a public interface of TF FE before